busy_beaver/toolbox.py

```diff
@@ -1,7 +1,27 @@
 from datetime import datetime, timedelta
+import json as _json
 
+from flask import Response
 import pytz
 
 
 def utc_now_minus(period: timedelta):
     return pytz.utc.localize(datetime.utcnow()) - period
+
+
+def make_response(
+    status_code: int = 200, *, headers: dict = {}, json: dict = {}, error: dict = {}
+):
```

Comment:
I do this for most of my projects. I'm not sure what good practice is, but it works well for me. Thoughts?

Reply:
I like it, makes it easy.

Reply:
Note: should set default values to `None`. If the value is still `None` inside the function, we can create an empty dictionary. This way can produce unexpected values as we are creating an object when this function is first read by the interpreter.

Currently, we are not seeing issues as the dictionary is not mutated, but it's better Python to change it. Do it in a separate PR so we can show that not everything is perfect and that we all screw up.